docs/components/tooltip.md

```diff
@@ -62,49 +62,42 @@ Here are the different options:
   </li>
 </ul>
 
-<div class="custom-tooltip">
-  <KTooltip placement="bottom" label="A label that appears on the bottom">
+<div class="tooltip-container">
+  <KTooltip placement="bottom" text="A tooltip that appears on the bottom.">
     <KButton>bottom</KButton>
   </KTooltip>
-  <KTooltip placement="top" label="A label that appears on the top">
-    <KButton>top</KButton>
+  <KTooltip placement="topEnd" text="A tooltip that appears on the top.">
+    <KButton>topEnd</KButton>
   </KTooltip>
-  <KTooltip placement="left" label="A label that appears on the left">
+  <KTooltip placement="left" text="A tooltip that appears on the left.">
     <KButton>left</KButton>
   </KTooltip>
-  <KTooltip placement="right" label="A label that appears on the right">
-    <KButton>right</KButton>
+  <KTooltip placement="bottomStart" text="A tooltip that appears on the bottom.">
+    <KButton>bottomStart</KButton>
   </KTooltip>
 </div>
 
-<style>
-.custom-tooltip {
-  display: flex !important;
-  justify-content: space-around !important;
-}
-</style>
-
 ```html
-<KTooltip placement="bottom" label="A label that appears on the bottom">
+<KTooltip placement="bottom" text="A tooltip that appears on the bottom.">
   <KButton>Sample Button</KButton>
 </KTooltip>
 ```
 
 ### positionFixed
 
-Use fixed positioning of the popover to avoid content being clipped by parental boundaries - defaults to `false`. See [`KPop` docs](popover.html#positionfixed) for more information.
+Use fixed positioning of the popover to avoid content being clipped by parental boundaries - defaults to `false`. See [KPop docs](/components/popover#positionfixed) for more information.
```

Comment:
question: any reason we default this to false instead of true?

Reply:
Changing to true

Reply:
This would be a breaking change. I'm asking, should we

Reply:
Based on discussion from the other thread yeah, we should